### DISCUSSION TITLE
Version of chromium

### DISCUSSION THREAD

**Discussion by rackedupkitty** [nwjs-ffmpeg-prebuilt/nwjs-ffmpeg-prebuilt#187] [Q&A]
When going through the releases, I am unable to tell what version of chromium each release is for, and from what I have heard it is important to have the right libffmpeg.so file for the correct version of chromium, which in my case is: 124.0.6367.243

**Comment by zkrige** [comment]
Hi!

We track NWJS builds - so whatever version of chromium NWJS is building, we will match that. You'll note our version numbers follow https://github.com/nwjs/nw.js. When they release a new version we run a pipeline that builds codecs for that version.

If you could figure out a way to extract chromium version and add it to our release notes, that would be very helpful going forwards

**Comment by MartinVonReichenberg** [comment]
> When going through the releases, I am unable to tell what version of chromium each release is for, and from what I have heard it is important to have the right libffmpeg.so file for the correct version of chromium, which in my case is: 124.0.6367.243

Hello, 

It is mentioned in the issue [here](https://github.com/nwjs-ffmpeg-prebuilt/nwjs-ffmpeg-prebuilt/issues/184).
`libffmpeg.so` version needs to match the Chromium version for some reason . . .

This is something which should be mentioned in README.

**Comment by MartinVonReichenberg** [comment]
More information about [CHROMIUM RELEASES HERE](https://chromiumdash.appspot.com/releases?platform=Linux)
